test/application_system_test_case.rb

```diff
@@ -5,6 +5,7 @@
 require "action_dispatch/system_testing/server"
 
 ActionDispatch::SystemTesting::Server.silence_puma = true
+Webdrivers::Chromedriver.required_version = "114.0.5735.90"
```

Comment:
We shouldn't need that either, we'll take care of the issue on main.

Reply:
opened a PR here https://github.com/Shopify/maintenance_tasks/pull/860